tests/unit_tests/test_db_handler.py

```diff
@@ -316,23 +316,37 @@ def test_separating_get_and_write(db):
         assert io.getOutputFile(fileNow, dirType="model", test=True).exists()
 
 
-def test_insert_files_db(db):
+def test_insert_and_get_files_db(db):
 
     logger.info("----Testing inserting files to the DB-----")
     logger.info(
         "Creating a temporary file in {}".format(io.getOutputDirectory(dirType="model", test=True))
     )
-    fileName = io.getOutputDirectory(dirType="model", test=True) / "test_file.dat"
-    with open(fileName, "w") as f:
-        f.write("# This is a test file")
+    outputDir = io.getOutputDirectory(dirType="model", test=True)
+    fileNames = ["test_file.dat", "test_file2.dat"]
```

Comment:
I assume you added this additional file in order to test your new function which exports a list of files. As I mentioned above, I don't think we need this function in the API, so the test can go back to it's original form.

Reply:
Since I will merge, for simplicity, both functions into one and they will deal both with a single file and a list of files, I will keep the test (and rather adapt it to the new function). Do you agree in this case to keep the test?

Reply:
Sure, that's fine

Reply:
Actually, now you will have to do both. You will have to have a test where you pass a list to `exportFileDB` and one where you pass just a string.